react/QuantitySelector.tsx

```diff
@@ -15,19 +15,24 @@ const MAX_ITEM_QUANTITY = 99999
 const CSS_HANDLES = ['quantitySelectorContainer'] as const
 
 type QuantitySelectorMode = 'default' | 'stepper'
+export type QuantitySelectorStepType = 'unitMultiplier' | 'singleUnit'
 
 interface Props {
   mode?: QuantitySelectorMode
+  quantitySelectorStep?: QuantitySelectorStepType
 }
 
-const QuantitySelector: VFC<Props> = ({ mode = 'default' }) => {
+const QuantitySelector: VFC<Props> = ({ mode = 'default', quantitySelectorStep = 'unitMultiplier' }) => {
   const { item, loading, onQuantityChange } = useItemContext()
   const handles = useCssHandles(CSS_HANDLES)
-
+  
   if (loading) {
     return <Loading />
   }
-
+  let unitMultiplier = item.unitMultiplier ?? undefined
```

Comment:
Why do you need the `?? undefined` part? 🤔  The `??` operator will only be useful if `item.unitMultiplier` is `undefined`, right?

Reply:
When i run `yarn lint` i get this error ![image](https://user-images.githubusercontent.com/55905671/125355712-b984dd00-e33b-11eb-858f-a0648a91ecb8.png)

Reply:
I remember why that `?? undefined`, if I don't add it then at build i get an error.

```
TS2322: Type 'number | null | undefined' is not assignable to type 'number | undefined'.
Type 'null' is not assignable to type 'number | undefined'.
[at-loader] ../../.../react/QuantitySelector.tsx:71:9
TS2322: Type 'number | null | undefined' is not assignable to type 'number | undefined'.
Type 'null' is not assignable to type 'number | undefined'
```

I changed to:   `const unitMultiplier = quantitySelectorStep === 'singleUnit' ? 1 : item.unitMultiplier ?? undefined
`